osu.Game/Rulesets/Mods/ModDifficultyAdjust.cs

```diff
@@ -26,7 +26,7 @@ public abstract class ModDifficultyAdjust : Mod, IApplicableToDifficulty
 
         public override Type[] IncompatibleMods => new[] { typeof(ModEasy), typeof(ModHardRock) };
 
-        [SettingSource("Drain Rate", "Override a beatmap's set HP.")]
+        [SettingSource("HP Drain", "Override a beatmap's set HP.")]
```

Comment:
What is the reason behind these name changes?

Reply:
To make them consistent with the beatmap detail area. But that place should be changed instead - the mods were already named correctly.